example/color/main.go

```diff
@@ -4,35 +4,63 @@ import (
 	"github.com/marcusolsson/tui-go"
 )
 
+// StyledBox is a Box with an overriden Draw method.
+// Embedding a Widget within another allows overriding of some behaviors.
+type StyledBox struct {
+	Style string
```

Comment:
This should probably be exported as a method rather than a field to keep the API consistent. I don't believe there are any exported fields in the current API (to my knowledge).

Reply:
Sorry, I scanned it too fast (this isn't part of the API). Ignore my comment.

Reply:
Yep, this is just internal for the example.